mods/ra/maps/allies-05a/rules.yaml

```diff
@@ -115,7 +115,7 @@ Colt:
 		Image: AGUN
 	Armament:
 		Weapon: MissionColt
-		LocalOffset: 432,150,-30, 432,-150,-30
+		LocalOffset: 520,100,450, 520,-150,450
```

Comment:
Why does this need an offset at all?  It's an invisible actor with no specific muzzle orientation.